src/uucore/src/lib/features/perms.rs

```diff
@@ -260,6 +260,27 @@ impl ChownExecutor {
                 }
             }
         } else {
+            if self.verbosity.level == VerbosityLevel::Verbose {
```

Comment:
Maybe move this into a function?

```
fn print_verbose(&self, path: &Path, uid: u32, gid: Option<u32>) {
    if self.verbosity.level == VerbosityLevel::Verbose {
        match gid {
            Some(gid) => {
                println!(
                    "ownership of {} retained as {}:{}",
                    path.quote(),
                    entries::uid2usr(uid).unwrap_or_else(|_| uid.to_string()),
                    entries::gid2grp(gid).unwrap_or_else(|_| gid.to_string()),
                );
            }
            None => {
                println!(
                    "ownership of {} retained as {}",
                    path.quote(),
                    entries::uid2usr(uid).unwrap_or_else(|_| uid.to_string()),
                );
            }
        }
    }
}

```

and call:
`print_verbose(path, uid, self.dest_gid.map(|_| meta.gid()));`